packages/node_modules/@webex/plugin-meetings/src/config.js

```diff
@@ -61,10 +61,10 @@ export default {
       verboseEvents: true
     },
     resolution: {
-      maxWidth: 1280,
-      maxHeight: 720,
-      idealWidth: 1280,
-      idealHeight: 720
+      maxWidth: 1920,
```

Comment:
are we sure it's fine to suddenly change this default from 720 to 1080 for everybody?